src/components/cards/WeatherCard.vue

```diff
@@ -164,36 +157,21 @@ export default {
 
       .condition
         position: absolute
-        left: 78%
+        left: 85%
         transform: translate(-50%, 0)
         display: grid
         place-items: center
 
         .condition-icon
-          font-size: 2em
-          color: var(--color)
```

Comment:
Any reason why you removed colors in the icons? Feel like it helped make the rest of the widget more coherent with the site.

Reply:
good catch, i'll add those back. SVG colors work weirdly so i'll have to do some research to fix this.

Reply:
<img width="589" alt="Screenshot 2023-12-28 at 5 41 52 PM" src="https://github.com/stevenson-space/shs/assets/31457361/4ae5db7b-4322-4b87-893b-a795fcfa74b3">